src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -384,16 +380,13 @@ public bool Dispatch(EventArgs eventArgs, string nativeEventPayload)
 						routedArgs.CanBubbleNatively = _canBubbleNatively;
 					}
 
-					if (_eventFilterManaged(args))
+					foreach (var handler in _invocationList)
```

Comment:
Would invocation list change during enumeration ?

Reply:
I didn't changed this code, but it's why there is a `_pendingInvocationList`: https://github.com/unoplatform/uno/pull/2054/files/940cff5808342a2ec7fa130c830bb914f6f184ad#diff-6f3c127d07fda497219e71a7d9b763a1R323